tests/lib.rs

```diff
@@ -293,12 +293,9 @@ fn write_random_sizes_out_of_sequence_with_gaps_and_overlaps() {
 fn cross_platform_check() {
     #[rustfmt::skip]
     static EXPECTED_HASHES: [[u8; 32]; 3] = [
-        [98, 4, 121, 14, 80, 44, 103, 243, 182, 251, 248, 171, 11, 254, 131, 103, 156, 178, 138,
-         217, 103, 136, 72, 247, 125, 195, 212, 138, 115, 116, 227, 114],
-        [167, 132, 31, 67, 233, 190, 215, 173, 208, 106, 242, 71, 1, 114, 195, 50, 127, 158, 143,
-         187, 72, 11, 106, 107, 65, 219, 4, 243, 46, 43, 29, 116],
-        [133, 35, 122, 193, 107, 25, 207, 120, 214, 149, 158, 221, 143, 155, 90, 97, 5, 77, 94,
-         52, 235, 252, 6, 150, 80, 188, 143, 146, 116, 221, 107, 107],
+        [90, 123, 178, 77, 189, 56, 250, 228, 43, 186, 33, 61, 74, 91, 212, 16, 157, 230, 227, 31, 132, 167, 178, 127, 44, 33, 184, 3, 80, 29, 195, 41],
```

Comment:
Why did it change, are we not using the same algorithm?

Reply:
Not exactly @jeanphilippeD :) We are updating it from rust_sodium's `secret_box::seal()` to `Cbc::encrypt_vec()` (which makes use of `Aes128`). Hence the change in hashes.